Fixed TS crane overlay's last frame hiding idle overlay

reaperrr opened the pull request:
TS crane overlay's last frame no longer overlaps idle overlay. `WithBuildingPlacedOverlay` is now only visible while playing the animation, rather than sticking around.

Fixes #9319.

reaperrr committed:
Hide WithBuildingPlacedOverlay after it has been played

Fixes #9319.

pchote commented:
Are all the other overlay traits going to have similar problems?

pchote commented on OpenRA.Mods.Common/Traits/Render/WithBuildingPlacedOverlay.cs:
```diff
@@ -35,6 +35,7 @@ public class WithBuildingPlacedOverlay : INotifyBuildComplete, INotifySold, INot
 	{
 		readonly Animation overlay;
 		bool buildComplete;
+		bool hide;
```
Might be a bit more consistent with other code to call this `visible` and invert the checks.

reaperrr replied:
Ok.

reaperrr commented:
> Are all the other overlay traits going to have similar problems?

Possibly. I'll take a look.

reaperrr committed:
Make repair overlay invisible when not repairing

reaperrr committed:
Fix TS repair depot idle overlay

This was an idle animation in the original, and this and "platform" cannot be displayed at the same time anyway.
Also increased Tick rate to match idle light.

reaperrr commented:
Fixed `WithRepairOverlay` as well. Looks like all other overlays have something comparable in place already.

Mailaender commented:
:+1: As a nice side effect this also makes the cloaking look better.

pchote commented:
Confirmed fixes :+1:.  Thanks.

pchote commented:
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/bf15da7fc151ed12b745d08315a37e74f1e07b04
